RELEASE_NOTES.md

```diff
@@ -6,6 +6,13 @@
 
 ## Upgrading
 
++ Battery power is no longer available through the `LogicalMeter`, but
+  through the `BatteryPool` (#338)
```

Comment:
```suggestion
* Battery power is no longer available through the `LogicalMeter`, but through the `BatteryPool` (#338)
```

(we are using mostly `*` in the file so far, and when the release notes are copied to the GitHub release, the markdown parsing is strange and keeps the new-lines, which makes it look weird)

Reply:
I don't like the idea of using `*` everywhere,  for second level also.  I like `+, -, *` in that order, to make it more readable.

I also think the Github line folding behaviour for release notes is intentional,  and it should be folded there to make the release notes easily readable, instead of having to read from left end of screen to the right.

But I guess "usually" is a good motivator as well.  So changed.

Reply:
I'm fine with using different chars for different levels, is just the inconsistency at the same level that makes it harder to read for me.

About the line length, maybe you are right, but I think it is a mistake, because the rendered should be able to decide which length is the most appropriate, if you have a very narrow screen, like a cell phone, then the lines will be automatically and artificially wrapped, making it very hard to read.